functions/_fk_pr.fish

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env fish
+
+function _fk_pr
+    if ! git rev-parse --show-toplevel > /dev/null 2> /dev/null
+        echo "Cannot open PR since current directory is not a git repo"
+        return 1
+    end
+
+    set curr_branch (git symbolic-ref --short HEAD)
+
+    if test curr_branch = "master"
+        echo "Cannot open PR for master branch"
+        return 2
+    end
+
+    set git_remote (git config --get remote.origin.url)
+    set repo_name (echo $git_remote | xargs basename | cut -d'.' -f1)
```

Comment:
Will cause issues with SSH vs HTTPS remotes. Check https://github.com/fantastic-kit/fantastic-kit/pull/31/

Reply:
not this line, the next one.